### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by lazartravica**
# Description

Please provide a detailed description of what was done in this PR

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [x] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

1. Prometheus namespace is now `polygon`, instead of `psdk`
2. Loadbot now reads from `LOADBOT_[ADDRESS]` env var instead of `PSDK_[ADDRESS]`

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [ ] I have tested this code
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯 

Polygon Edge here we go 🚀

**Review by Kourin1996 — APPROVED** [review]

**Review by 0xAleksaOpacic — APPROVED** [review]
Well done!

**Review by munna0908 — APPROVED** [review]
LGTM 🙏

File: .github/workflows/main.yml
Comment (3Y215) [inline .github/workflows/main.yml:1]: Test up

**Pull request merged**